packages/axelar-wasm-std/src/event.rs

```diff
@@ -0,0 +1,38 @@
+use serde::Serialize;
+use serde_json::to_string;
+
+pub fn attribute_value<T>(v: &T) -> Result<String, serde_json::error::Error>
+where
+    T: ?Sized + Serialize,
+{
+    let json = to_string(v)?;
+    // using strip_* instead of trim_matches because the latter trims any number of quotes instead of just one
+    let json = json.strip_prefix('"').unwrap_or(&json);
+    Ok(json.strip_suffix('"').unwrap_or(json).to_string())
```

Comment:
```suggestion
    let json = json
        .strip_prefix('"')
        .unwrap_or(&json)
        .strip_suffix('"')
        .unwrap_or(&json);
    Ok(json.to_string())
```
I'm detinitely nitpicking but I feel this is clearer cuz each statement has a very explicit purpose.